geoportailv3/static/js/search/searchdirective.js

```diff
@@ -197,17 +198,29 @@ app.SearchDirectiveController =
   var POIBloodhoundEngine = this.createAndInitPOIBloodhound_(
       ngeoCreateGeoJSONBloodhound, searchServiceUrl);
 
-  /** @type {Bloodhound} */
-  var LayerBloodhoundEngine = this.createAndInitLayerBloodhound_();
-
-  /** @type {Bloodhound} */
-  var BackgroundLayerBloodhoundEngine = this.createAndInitLayerBloodhound_();
+  /** @type {Fuse} */
+  var LayerFuseEngine =
```

Comment:
This is minor, but `LayerFuseEngine` is not a type, so it's name should not start with a capital layer.